default/hello-world/src/panels/HelloWorldPanel.ts

```diff
@@ -27,7 +27,7 @@ export class HelloWorldPanel {
 
     // Set an event listener to listen for when the panel is disposed (i.e. when the user closes
     // the panel or when the panel is closed programmatically)
-    this._panel.onDidDispose(this.dispose, null, this._disposables);
+    this._panel.onDidDispose(() => this.dispose(), null, this._disposables);
```

Comment:
```suggestion
    this._panel.onDidDispose(this.dispose, this, this._disposables);
```

Reply:
`this.panel.onDidDispose(() => this.dispose(), null, this.disposables);`
 ------------------------------ or ---------------------------------------
`this._panel.onDidDispose(this.dispose, this, this._disposables); `      

Both solve the bug.

Reply:
Thanks for the suggestion @kahlkevin! I too confirmed that both worked but ultimately ended up going with:

```ts
this.panel.onDidDispose(() => this.dispose(), null, this.disposables);
```

since that's the convention that can be found in the [official VS Code webview sample](https://github.com/microsoft/vscode-extension-samples/blob/main/webview-sample/src/extension.ts#L97)